### PR TITLE
Fix for missing repositories

### LINKED ISSUE #35: Explore with `fusion-orchestrate` does not include all repositories
#### Problem/Rationale

Some repositories, like `fusion-plugin-font-loader-react` do not appear to be included in any of the `fusion-orchestrate` commands when they should be.

#### Solution/Change/Deliverable

Explore why this may be occurring and rectify the issue(s).

Possible areas to check:
* Repository settings mismatch
* User permissions for missing repos
* GitHub API in `fusion-orchestrate`

### PR DESCRIPTION
**Pull request by alxmyth**
Fixes #35

> Some repositories, like fusion-plugin-font-loader-react do not appear to be included in any of the fusion-orchestrate commands when they should be.

### CODE REVIEW

File: src/repos.js [hunk 1]
```diff
@@ -4,9 +4,22 @@ const api = require('./api');
...
 
 const getForOrg = async org => {
-  const {data} = await api.repos.getForOrg({
-    org: org,
-  });
+  let pageNum = 1;
+  let results = [];
+  let isEmpty = false;
+  while (!isEmpty) {
+    const {data} = await api.repos.getForOrg({
```
Comment (alxmyth) [inline src/repos.js:11]: Defaults to [30 results per page](https://octokit.github.io/rest.js/#api-Issues-getForOrg) and we now have over 30 repositories in the _fusionjs_ org.

File: src/repos.js [hunk 2]
```diff
@@ -4,9 +4,22 @@ const api = require('./api');
...
+      page: pageNum,
+    });
+    results.push(data);
+
+    pageNum++;
+    isEmpty = data.length === 0;
+  }
+
+  const data = [].concat.apply([], results);
+  console.log(data.map(d => d.name));
```
Comment (KevinGrandon) [inline src/repos.js]: Should we remove console.log, or hide it under some verbose flag?

**Review by KevinGrandon — DISMISSED** [review]
Nice catch, thanks!

File: src/repos.js (hunk 2 above)
Comment (alxmyth) [reply src/repos.js]: I'll remove it

**Pull request merged**